include/deal.II/distributed/solution_transfer.h

```diff
@@ -148,6 +149,29 @@ namespace parallel
      * @endcode
      *
      *
+     * <h3>Note on usage with hp::DoFHandler</h3>
+     *
+     * If an object of the hp::DoFHandler class is registered with an
+     * instantiation of this parallel::distributed::SolutionTransfer
+     * class, the following requirements have to be met:
+     * <ul>
+     *   <li>
+     *     The hp::DoFHandler needs to be explicitely mentioned
+     *     in the parallel::distributed::SolutionTransfer type, i.e.:
+     *     @code
+     *     parallel::distributed::SolutionsTransfer<dim, VectorType,
+     *       hp::DoFHandler<dim, spacedim>> sol_trans(hp_dof_handler);
+     *     @endcode
+     *   </li>
+     *   <li>
+     *     The transfer of the <tt>active_fe_index</tt> of each cell
```

Comment:
Maybe mention briefly what that transfer does? I thought that if I do refinement only, I can manually keep FE-indices on child cells to be the same as on the parent. That should be enough for this transfer to work, right?

Reply:
Repartitioning of the mesh is scheduled in the `p::d` case after refinement, which changes the ownership of cells. So we have to also send additional information of the cell, i.e. the `active_fe_index`, as well.

I extended the comment on that.

Reply:
Could add a brief note on requirements in terms of FE indices (all child cells have the same index as parent) and also mention the coarsening and least dominating FE among child cells.

Reply:
Will do!

This information is actually also missing in the documentation of the ActiveFEIndicesTransfer class. I"ll add a similar one there as well.